kubeflow/instance/kustomize/iap-ingress/kustomization.yaml

```diff
@@ -3,6 +3,7 @@ kind: Kustomization
 patchesStrategicMerge:
 - iap-ingress-config.yaml
 - service-accounts.yaml
+- service.yaml
```

Comment:
nit: can we add a comment here with short explanation of why we want to add this patch and linking to the issue?